src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx

```diff
@@ -6,15 +6,58 @@
 import { ImplementedChange } from 'shared';
 import { datePipe, emDashPipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
 import { routes } from '../../utils/routes';
-import { Link, ListItem, List, Tooltip, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Link, ListItem, List, Tooltip, Typography, TooltipProps, tooltipClasses } from '@mui/material';
 import PageBlock from '../../layouts/PageBlock';
+import { useLayoutEffect, useState } from 'react';
 
 interface ImplementedChangesListProps {
   changes: ImplementedChange[];
   overallDateImplemented?: Date;
 }
 
 const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes, overallDateImplemented }) => {
+  const [bodyWidth, setBodyWidth] = useState<number>(window.document.body.offsetWidth);
+
+  window.document.body.addEventListener('resize', () => {
+    setBodyWidth(window.document.body.offsetWidth);
```

Comment:
I tried resetting the width, and it didn't seem to call setBodyWidth in here. So bodyWidth was fixed at loading time. Not sure if that's what you intended.